ui/src/global.d.ts

```diff
@@ -0,0 +1 @@
+import "vitest-fetch-mock";
```

Comment:
question: is this supposed to be under `src`?

Reply:
Ah thanks this is actually no longer required as I fixed the typescript config in my [previous PR](https://github.com/canonical/identity-platform-admin-ui/commit/7781bcb13d2ea71c0b898c8ff47131334fd7df9e#diff-3ec3dd66f196bbc443564513e0afea83c0b280044551e1d29715f943a18147d3). Global definitions do need to be in the root, but don't get included in the build as they're just type annotations.